tests/coprocessor/test_select.rs

```diff
@@ -103,161 +401,83 @@ impl Store {
+    let mut store = Store::new(engine.clone());
 
-    for col in group_by {
-        let mut item = ByItem::new();
-        let mut expr = Expr::new();
-        expr.set_tp(ExprType::ColumnRef);
-        expr.mut_val().encode_i64(col).unwrap();
-        item.set_expr(expr);
-        sel.mut_group_by().push(item);
+    store.begin();
+    for &(ref id, ref name, ref count) in vals {
+        store.insert_into(&tbl.table)
+            .set(tbl.id, id.clone())
+            .set(tbl.name, name.clone())
+            .set(tbl.count, count.clone())
+            .execute();
     }
+    store.commit();
 
-    sel
-}
-
-fn prepare_sel(store: &mut Store,
-               ti: &TableInfo,
-               limit: Option<i64>,
-               desc: Option<bool>,
-               aggrs: Vec<Expr>,
-               group_by: Vec<i64>)
-               -> Request {
-    let mut sel = build_basic_sel(store, limit, desc, aggrs, group_by);
-    sel.set_table_info(ti.as_pb_table_info());
-
-    let mut req = Request::new();
-    req.set_tp(REQ_TYPE_SELECT);
-    req.set_data(sel.write_to_bytes().unwrap());
-
-    let mut range = KeyRange::new();
-
-    let mut buf = Vec::with_capacity(8);
-    buf.encode_i64(i64::MIN).unwrap();
-    range.set_start(table::encode_row_key(ti.t_id, &buf));
-
-    buf.clear();
-    buf.encode_i64(i64::MAX).unwrap();
-    range.set_end(table::encode_row_key(ti.t_id, &buf));
-    req.set_ranges(RepeatedField::from_vec(vec![range]));
-    req
-}
+    let runner = EndPointHost::new(engine);
+    let mut end_point = Worker::new("test select worker");
+    end_point.start_batch(runner, 5).unwrap();
 
-// This function will insert `count` rows data into table.
-// each row contains 3 column, first column primary key, which
-// id is 1. Second column's type is a varchar, id is 3, value is
-// `varchar:$((handle / 2))`. Third column's type is long, id is 4,
-// value is the same as handle.
-// For example:
-// 1 varchar:0 1
-// 2 varchar:1 2
-// 3 varchar:1 3
-// ...
-// 9 varchar:4 9
-// 10 varchar:5 10
-fn initial_data(count: i64) -> (Store, Worker<RequestTask>, TableInfo) {
-    let engine = Arc::new(engine::new_engine(Dsn::RocksDBPath(TEMP_DIR), DEFAULT_CFS).unwrap());
-    let mut store = Store::new(engine.clone());
-    let ti = TableInfo {
-        t_id: 1,
-        c_types: vec![TYPE_VAR_CHAR, TYPE_LONG],
-        c_ids: vec![3, 4],
-        idx_off: vec![0],
-        i_ids: vec![5],
-    };
-    prepare_table_data(&mut store, &ti, count);
-
-    let end_point = EndPointHost::new(engine);
-    let mut worker = Worker::new("test select worker");
-    worker.start_batch(end_point, 5).unwrap();
-    (store, worker, ti)
+    (store, end_point)
 }
 
 #[test]
 fn test_select() {
-    let count = 10;
-    let (mut store, mut end_point, ti) = initial_data(count);
-    let req = prepare_sel(&mut store, &ti, None, None, vec![], vec![]);
+    let data = vec![
```

Comment:
The data is still obscure with Datum initializer, how about define the data in pure string, and use a function to parse the string into datum array, the column type can be used to determine datum type.